src/Servers/HttpSys/src/NativeInterop/HttpApi.cs

```diff
@@ -25,7 +26,7 @@ internal static unsafe class HttpApi
         internal static extern uint HttpReceiveClientCertificate(SafeHandle requestQueueHandle, ulong connectionId, uint flags, byte* pSslClientCertInfo, uint sslClientCertInfoSize, uint* pBytesReceived, SafeNativeOverlapped pOverlapped);
 
         [DllImport(HTTPAPI, ExactSpelling = true, CallingConvention = CallingConvention.StdCall, SetLastError = true)]
-        internal static extern uint HttpReceiveHttpRequest(SafeHandle requestQueueHandle, ulong requestId, uint flags, HTTP_REQUEST* pRequestBuffer, uint requestBufferLength, uint* pBytesReturned, SafeNativeOverlapped pOverlapped);
+        internal static extern uint HttpReceiveHttpRequest(SafeHandle requestQueueHandle, ulong requestId, uint flags, HTTP_REQUEST* pRequestBuffer, uint requestBufferLength, uint* pBytesReturned, NativeOverlapped* pOverlapped);
```

Comment:
Why change this? How is the object lifetime safety being managed?

Reply:
Because we want to avoid allocation this per operation when it can be re-used for all accepts

Reply:
Oh, you're depending on changes from one of our other PRs. I'll come back to this when I've had a chance to review that one first.

Reply:
I'm gonna move this change to the other PR